src/stories/header.css

```diff
@@ -0,0 +1,32 @@
+.storybook-header {
```

Comment:
do we need these CSS files? @kumuditha-udayanga

Reply:
No, these are related to the sample stories. Shall I remove those components too? @anjula-sack